subsys/net/openthread/Kconfig

```diff
@@ -19,12 +19,47 @@ config NORDIC_SECURITY_PROMPTLESS
 choice OPENTHREAD_SECURITY
 	depends on !OPENTHREAD_SECURITY_INTERNAL
 
+config OPENTHREAD_NRF_SECURITY_PSA_CHOICE
+	bool "nRF Security with PSA crypto enabled"
+	select OPENTHREAD_NRF_SECURITY_PSA
+
 config OPENTHREAD_NRF_SECURITY_CHOICE
 	bool "nRF Security"
 	select OPENTHREAD_NRF_SECURITY
 
 endchoice
 
+config OPENTHREAD_NRF_SECURITY_PSA
+	bool
+	select EXPERIMENTAL
+	select NRF_SECURITY
+	select MBEDTLS_ENABLE_HEAP
+	select MBEDTLS_PSA_CRYPTO_STORAGE_C if !BUILD_WITH_TFM
+	select PSA_NATIVE_ITS if !BUILD_WITH_TFM
+	select PSA_WANT_ALG_RSA_OAEP
+	select PSA_WANT_ALG_HKDF
+	select PSA_WANT_ALG_CCM
+	select PSA_WANT_ALG_CMAC
+	select PSA_WANT_ALG_HMAC
+	select PSA_WANT_ALG_ECB_NO_PADDING
+	select PSA_WANT_ALG_SHA_256
+	select PSA_WANT_ALG_JPAKE
```

Comment:
This may be a candidate for a followup task, but it may be helpful to enable TLS/JPAKE bits only if the commissioner/joiner is enabled to save flash on devices that don't need it.

Reply:
Yes, we've discussed that earlier and decided to proceed with this approach for the experimental support as there are some dependencies in openthread that require ecjpake support whenever DTLS is used (so we cannot simply make it dependent only on commissioner/joiner, because COAPS uses DTLS too). We will work on these dependencies as next step.